packages/manager/modules/pci/src/projects/project/instances/instance/edit/edit.routing.js

```diff
@@ -28,19 +37,26 @@ export default /* @ngInject */ ($stateProvider) => {
               )
               .then(({ data }) => data),
           })
-
           .then(
             ({ region, quota }) =>
               new Datacenter({
                 ...region,
                 quota,
               }),
           ),
+
+      excludeCategories: /* @ngInject */ (pciFeatures) => {
```

Comment:
duplicate: same method id defined in packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js. Can you reuse it?

Reply:
Yes i also saw that, but i prefer to have a separate categories to have the ability to deals both easly.